Chummer/Backend/Uniques/Spell.cs

```diff
@@ -849,6 +849,21 @@ private IEnumerable<Improvement> RelevantImprovements(Func<Improvement, bool> fu
                             yield return objImprovement;
                         break;
                     case Improvement.ImprovementType.SpellCategory:
+                        // We need to do some checking to make sure this is the most powerful foci before we add it in
+                        if (objImprovement.ImproveSource == Improvement.ImprovementSource.Gear )
```

Comment:
Is this the best way to see if the SpellCategory improvement is a focus? Is there a narrower category I should be using?

Reply:
Yeah, there's not really anything cuter than that. You could ghetto something with GetSourceName, but it'd be a kludge.